src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -691,8 +708,9 @@ void GeneralPage::initLayout(GeneralPageView &layout)
         [](auto args) {
             return args.index;
         },
-        false);
-    layout.addCheckbox("Combine multiple bit tips into one", s.stackBits);
+        false, "Combine consecutive timeout messages into a single message.");
+    layout.addCheckbox("Combine multiple bit tips into one", s.stackBits, false,
+                       "Combine consecutive bit tips into a single message.");
```

Comment:
```suggestion
    layout.addCheckbox("Combine multiple bit tips into one", s.stackBits, false,
                       "Combine consecutive cheermotes into a single one.");
```
This setting is about combining bits from a single message into a single emote.

Reply:
There needs to be emphasis that this concerns multiple cheermotes in a single message and not single cheermotes in multiple messages. How about "Combine consecutive cheermotes (sent in a single message) into one cheermote"?